platforms/extensibility/plugin-development/src/main/java/org/gradle/plugin/devel/tasks/ValidatePlugins.java

```diff
@@ -109,9 +109,10 @@ public void validateTaskClasses() throws IOException {
             });
         getWorkerExecutor().await();
 
-        List<? extends Problem> problems = ValidationProblemSerialization.parseMessageList(new String(Files.readAllBytes(getOutputFile().get().getAsFile().toPath())));
+        Long operationId = Long.valueOf(new String(Files.readAllBytes(getOutputFile().get().getAsFile().toPath())));
```

Comment:
We still need the file here. Now, we only store the current operation ID in there. The daemon can use it to look up which problems were reported from a validation worker.